src/Metal/MTLResourceStatePassSampleBufferAttachmentDescriptorArray.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace Metal {
+
+#if !TVOS
+
+	public partial class MTLResourceStatePassSampleBufferAttachmentDescriptorArray {
+
+		public MTLResourceStatePassSampleBufferAttachmentDescriptor this[nuint i] {
```

Comment:
where `[Get|Set]Object` introduced in the same OS version as the type ?

Reply:
Yes, new class, new methods.